test_scripts/AppServices/OnAppServiceData/009_M2M_service_not_published_flow.lua

```diff
@@ -0,0 +1,97 @@
+---------------------------------------------------------------------------------------------------
+--  Precondition: 
+--  1) Application 1 with <appID> is registered on SDL.
+--  2) Application 2 with <appID2> is registered on SDL.
+--  3) Specific permissions are assigned for <appID> with PublishAppService
```

Comment:
Might be better to say App Service Provider permissions. Its not using PublishAppService in this test but it is using OnAppServiceData